docs/sources/next/extensions/explore.md

```diff
@@ -185,15 +161,7 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
     </a>
     <a href="https://github.com/pmalhaire/xk6-mqtt" target="_blank" class="nav-cards__item nav-cards__item--guide">
         <h4>xk6-mqtt</h4>
-        <p>mqtt extension</p>
-    </a>
-    <a href="https://github.com/ydarias/xk6-nats" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements
https://github.com/ydarias/xk6-nats/issues/15
https://github.com/ydarias/xk6-nats/issues/14